packages/radix-ui-themes/src/theme-panel.css

```diff
@@ -32,7 +32,7 @@
 .rt-ThemePanelSwatchInput {
   outline-offset: 2px;
 
-  &:where(:checked) {
+  &:where([data-checked]) {
```

Comment:
I can't figure out why, but even on `main` the theme panel works weird on the /test-appearance page

CSS is not picking up `:checked` until there's a re-render 🤷